provider/k8s.go

```diff
@@ -1164,6 +1165,15 @@ func (iter *FileStoreFeatureIterator) Next() bool {
 
 func castToTimestamp(timestamp interface{}) (time.Time, error) {
 	if ts, ok := timestamp.(time.Time); !ok {
+		// TODO: this is a temporary fix for the timestamp issue
```

Comment:
Your call, but I'd do a:
switch timestamp.(type) {
case time.Time:
case string:
default:
}

Reply:
i agree w/ simba's, easier to read